### PR TITLE
Fix bug 1113274: Add CONTRIBUTING file, slim down README.

### PR DESCRIPTION
**Pull request by Osmose**

### CODE REVIEW

**Comment by Osmose** [comment]
@pmclanahan, @ckprice r?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+and reference guide to contributing as a web developer to Mozilla.
+
+
+## Reporting Issues
+
+We use [Bugzilla][] to track issues and bugs for Project Lorax. Any issues
+with the website should be filed under the Shape of the Web component in the
+Websites product.
+
+[Bugzilla]: https://bugzilla.mozilla.org/buglist.cgi?product=Websites&component=Shape of the Web&resolution=---
```
Comment (pmclanahan) [inline CONTRIBUTING.md]: For some reason it's not parsing this correctly. Might be the spaces in the URL.

**Comment by pmclanahan** [comment]
Quick look is very good! Thanks! I can read it more in-depth on Monday when I'm not on PTO. But feel free to merge sooner if you and Cory wish and we can make more changes later.

**Comment by Osmose** [comment]
Fixed the warnings that @ckprice encountered and the whitespace issue in the URL.

**Comment by ckprice** [comment]
Ran tutorial, r+! http://cl.ly/image/28440i3S2O0K

@Osmose I can merge if you give me write access.

Thanks!

**Pull request merged**